app/stac_api/views_mixins.py

```diff
@@ -167,3 +171,28 @@ def perform_destroy(self, instance):
             raise serializers.ValidationError(
                 _(f'Deleting {instance.__class__.__name__} with {child_name} not allowed')
             ) from None
+
+
+def patch_cache_settings_by_update_interval(response, update_interval):
+    max_age = get_dynamic_max_age_value(update_interval)
```

Comment:
this can be `-1` as well right? what happens in this case?

Reply:
@boecklic In this case there is no cache-control header patching here and it is done via the cache middleware, see https://github.com/geoadmin/service-stac/blob/3b7c4d1ed106679a129a840433ba961246fca87d/app/middleware/cache_headers.py#L49